_data/projects/nunit.yml

```diff
@@ -2,10 +2,17 @@ name: NUnit
 desc: NUnit is the most popular and widely used unit test framework for .NET
 site: https://github.com/nunit/nunit
 tags:
-- ".NET"
+- NUnit
+- .NET
+- .NET Core
```

Comment:
No worries! Just keep an eye on the CI results - generally speaking, if a CI check fails, then your PR won't be approved here.